src/test/java/com/puppycrawl/tools/checkstyle/AbstractPathTestSupport.java

```diff
@@ -110,4 +355,58 @@ protected static String toLfLineEnding(String text) {
         return text.replaceAll(CR_FOLLOWED_BY_LF_REGEX, "");
     }
 
+    /**
+     * Validate that all input files do not exceed specified max length.
+     *
+     * @param theFiles list of files
+     * @throws CheckstyleException if file length is more than max length
+     * @throws IOException if I/O exception occurs while reading
+     */
+    protected static void validateFileLength(
```

Comment:
Why not just use https://checkstyle.org/config_sizes.html#FileLength ?

Reply:
I didn't notice that check earlier, updated it to use it now, also please restart appveyor

Reply:
Restarted but other jobs are failing.

Reply:
Yup, just a moment, I filtered for `.java` files, adding other extensions too.